dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/AuthorityEntryLinkRepository.java

```diff
@@ -54,6 +55,7 @@ public HALResource wrapResource(AuthorityEntryRest model, String... rels) {
         return new AuthorityEntryResource(model);
     }
 
+    @PreAuthorize("hasAuthority('AUTHENTICATED')")
```

Comment:
I'm not sure that this should be really restricted to authenticated user. It could be needed in the public UI to display richer information for metadata linked to an authority